Source/JavaScriptCore/b3/B3ValueRep.h

```diff
@@ -129,7 +129,22 @@ class ValueRep {
     ValueRep(Kind kind)
         : m_kind(kind)
     {
-        ASSERT(kind == WarmAny || kind == ColdAny || kind == LateColdAny || kind == SomeRegister || kind == SomeRegisterWithClobber || kind == SomeEarlyRegister || kind == SomeLateRegister);
+        ASSERT(kind == WarmAny
+            || kind == ColdAny
+            || kind == LateColdAny
+            || kind == SomeRegister
+            || kind == SomeRegisterWithClobber
+            || kind == SomeEarlyRegister
+            || kind == SomeLateRegister
+#if USE(JSVALUE32_64)
+            || kind == SomeRegisterPair
```

Comment:
Do we really need this many variants?

Reply:
Not at the moment, no. I think it's easier to remove those we don't end up using than adding individual variants as they are needed though.

FWIW the variants are already there, it's just that this assert had not been updated to account for them. If we want to remove them and add them back as needed, I'd submit a separate PR for that.